pkg/storage/replica_proposal.go

```diff
@@ -404,6 +405,7 @@ func addSSTablePreApply(
 	} else {
 		ingestPath := path + ".ingested"
 
+		canIngestRaftFile := false
```

Comment:
The name of this variable was mildly confusing to me initially. We're going to ingest regardless, what you're checking is whether we can do the fast path ingestion by hard linking to the sideloaded file. I don't have a better suggestion, though.

Reply:
hm. The _content_ is always from raft, but the _file_ passed to ingest may or may not be raft's, which is what I was trying to say there.

Reply:
renamed to `canLinkToRaftFile`